auth/MobileController.js

```diff
@@ -74,10 +73,15 @@ router.get('/redirecturi', function(req, res) {
       // Save to the config to scoutuser data
       console.log(`Authorized: ${pocketUserId}/${pocketUserAccessToken}`);
       await database.processScoutUser(pocketUserId, pocketUserAccessToken);
-      const result = {
-        userid: pocketUserId
-      };
-      res.status(200).send(JSON.stringify(result));
+
+      // Send custom protocol redirect for use by Scout mobile app
+      const location = `mozilla-skout://user/${pocketUserId}`;
```

Comment:
Do we need to have a different path now for iOS vs. Android?  Should we file a follow up for the future to have a separate API for iOS vs. Android Auth?  Assuming we don't need this custom protocol redirect for Android.